data/campaigns/Northern_Rebirth/scenarios/03_To_the_Mines.cfg

```diff
@@ -16,7 +16,7 @@
     [story]
         [part]
             music=traveling_minstrels.ogg
-            story= _ "Hoping to bypass the monsters which seemed to lurk at every corner in the caves, the humans and the dwarves returned to the surface hoping to speedily reach the mines. However, once they left the caves they found that they faced a new threat."
+            story= _ "Hoping to bypass the monsters which seemed to lurk at every corner in the caves, the humans and the dwarves returned to the surface in hope to speedily reach the mines. However, once they left the caves they found that they faced a new threat."
```

Comment:
The original reads better to me - this change sounds awkward, not even sure I would consider it grammatically correct.

Reply:
Changed it back.